tests/cypress/page-object/worflow.dashboard.page.ts

```diff
@@ -0,0 +1,15 @@
+import { BasePage } from './base.page'
+import { siteHomePage } from './site.home.page'
+
+class WorkflowDashboardPage extends BasePage {
+    elements = {
+        publishAll: "[class *= 'bbar'] [class*='button-bypassworkflow']",
+    }
+
+    clickPublishAll() {
+        this.getIframeBody().find(this.elements.publishAll).click()
```

Comment:
Do we want to check if there is something to publish?
Or just hard publish without going through the workflow dashboard?

Reply:
the purpose is not to test the workflow, it's just to publish the site to see that sitemap is working